rio_cogeo/scripts/cli.py

```diff
@@ -84,9 +85,9 @@ def cogeo(
 
     config = dict(
         NUM_THREADS=threads,
-        BIGTIFF="IF_SAFER",
-        GDAL_TIFF_INTERNAL_MASK=True,
-        GDAL_TIFF_OVR_BLOCKSIZE=block_size,
+        BIGTIFF=os.environ.get("BIGTIFF", "IF_SAFER"),
```

Comment:
@vincentsarago `BIGTIFF` is a dataset creation option, not a config option, so it must be passed to `rasterio.open` when we create the output file.

Reply:
great, so should we add it to the `profiles` ?

Reply:
Yep.